pytumblr/__init__.py

```diff
@@ -488,6 +488,23 @@ def edit_post(self, blogname, **kwargs):
         valid_options = ['id'] + self._post_valid_options(kwargs.get('type', None))
         return self.send_api_request('post', url, kwargs, valid_options)
 
+    @validate_blogname
+    def notes_post(self, blogname, id, **kwargs):
```

Comment:
I think the name should be either just `notes` or `post_notes`. We can only have notes on posts so the first name would be preferred. Thoughts?

Reply:
That makes sense, can't remember why I had picked notes_post for the name before.  I have included a commit to change the name to notes.